bolt/lib/Profile/StaleProfileMatching.cpp

```diff
@@ -206,13 +213,30 @@ class StaleMatcher {
         CallHashToBlocks[CallHashes[I]].push_back(
             std::make_pair(Hashes[I], Block));
     }
+    this->IndexToBBPseudoProbes = IndexToBBPseudoProbes;
+    this->BBPseudoProbeToBlock = BBPseudoProbeToBlock;
+    this->YamlBFGUID = YamlBFGUID;
```

Comment:
As we only perform pass-through assignment with these members, would it make sense to set them in the constructor or simply expose as public?

Reply:
Made some changes to the stale matcher class in response to this - added a constructor and some setters. Might want to follow up in a separate PR and make similar NFC changes to OpHashToBlocks and CallHashToBlocks. Let me know what you think.

Reply:
Yes, that looks good.